src/app/docs/description-list/description-list-docs.component.html

```diff
@@ -0,0 +1,90 @@
+<sky-docs-demo-page
+  moduleName="SkyDescriptionListModule"
+  moduleSourceCodePath="src/app/public/modules/description-list/"
+  pageTitle="Description list"
+>
+  <sky-docs-demo-page-summary>
+    Description lists display lists of terms and pairs each term with a description.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo
+    heading="Name-value pair demo"
+  >
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+      >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          initialValue="vertical"
+          propertyName="orientation"
+          [choices]="layoutChoices"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+    <sky-description-list
+      mode="standard"
+      [orientation]="demoSettings.orientation"
+    >
+      <sky-description-list-content *ngFor="let item of nameValueItems">
+        <sky-description-list-term>
+          {{item.term}}
+        </sky-description-list-term>
+        <sky-description-list-description>
+          {{item.description}}
+        </sky-description-list-description>
+      </sky-description-list-content>
+    </sky-description-list>
+  </sky-docs-demo>
+
+  <sky-docs-demo
+    heading="Term-description list demo"
+  >
+    <sky-description-list
+      mode="longDescription"
+    >
+      <sky-description-list-content *ngFor="let item of longDescriptionItems">
+        <sky-description-list-term>
+          {{item.term}}
+        </sky-description-list-term>
+        <sky-description-list-description>
+          {{item.description}}
+        </sky-description-list-description>
+      </sky-description-list-content>
+    </sky-description-list>
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <p>
+        Don't use colons after description list terms.
+      </p>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux-indicators/docs/alert">Alert</a></li>
+          <li><a href="https://developer.blackbaud.com/skyux/components/toolbar">Toolbar</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+  </sky-docs-design-guidelines>
+
+  <!-- <sky-docs-code-examples>
+    <sky-docs-code-example
+      heading="Name-value pair"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/description-list/name-value/"
```

Comment:
Is this path correct? If so, should we change it to reflect that we aren't using "name-value" for this component? Seems like to me.

Reply:
It doesn't exist yet because we don't have demos. I'll just remove the commented code.

Reply:
Done.